modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java

```diff
@@ -515,18 +761,27 @@ public ClusterBuilder end() {
             return parent;
         }
 
-        private TestTable build() {
-            TableDescriptorImpl tableDescriptor = new TableDescriptorImpl(columns, distribution);
+        private List<CatalogCommand> build() {
+            List<CatalogCommand> commands = new ArrayList<>(1 + indexBuilders.size());
 
-            List<IgniteIndex> indexes = indexBuilders.stream()
-                    .map(idx -> idx.build(tableDescriptor))
-                    .collect(Collectors.toList());
+            commands.add(
+                    CreateTableCommand.builder()
+                            .schemaName("PUBLIC")
```

Comment:
Perhaps we can expose schema name via builder and assign default value to `PUBLIC` (Catalog::DEFAULT_SCHEMA_NAME would be better). What do you think?

Reply:
honestly, I see no point in using different schemas... It's just a namespace that is used during name resolution only